_el/exercises.md

```diff
@@ -336,172 +204,52 @@ piece of code with the output it produces".
+
+Οι [[Basu2015](#CITE)] έδωσαν στους εκπαιδευόμενους μια σειρά από λύσεις δοκιμαστικών τεστ, αλλά οι μαθητές έπρεπε να ξεκλειδώσουν το καθένα απαντώντας σε ερωτήσεις σχετικά με την αναμενόμενη συμπεριφορά τους πριν τους επιτραπεί να το εφαρμόσουν στις προτεινόμενες από αυτές λύση. Για παράδειγμα, έστω ότι οι μαθητές γράφουν μια λειτουργία για να βρουν το μεγαλύτερο γειτονικό ζευγάρι αριθμών σε μια λίστα. Πριν να τους επιτραπεί η χρήση των δοκιμών που σχετίζονται με αυτό το πρόβλημα, οι μαθητές θα πρέπει πρώτα να επιλέξουν τη σωστή απάντηση στο ερώτημα "Ποιό είναι το αποτέλεσμα της κλήσης `largestPair(4, 3, -1, 5, 3, 3)`;" (Η σωστή απάντηση είναι «`(5, 3)`.) Σε ένα πανεπιστημιακό μάθημα 1300 ατόμων, η μεγάλη πλειοψηφία των μαθητών επέλεξε να επικυρώσει την κατανόησή των δοκιμαστικών τέστ με αυτόν τον τρόπο πριν να επιχειρήσουν να λύσουν τα προβλήματα, με αποτέλεσμα στη συνέχεια να κάνουν λιγότερες ερωτήσεις και να εκφράσουν λιγότερη σύγχυση σχετικά με το περιεχόμενο των προβλημάτων.
+
+Είναι πράγματι πολύ δελεαστικό να χρησιμοποιήσει κανείς τα αυτοματοποιημένα εργαλεία ελέγχου για τη βαθμολόγηση του κώδικα των μαθητών. Ωστόσο, [[Nutb2016](#CITE)] αρχικά δεν βρήκε συσχετισμό μεταξύ των βαθμών που παρέχονται από ανθρώπινη βαθμολόγηση και των παραβιάσεων που προέρχονται από τα αυτόματα εργαλεία. Μερικές φορές αυτό οφείλεται στο γεγονός ότι οι μαθητές παραβίασαν ένα μόνο κανόνα πολλές φορές (χάνοντας έτσι περισσότερους βαθμούς από ό,τι θα έπρεπε) ενώ άλλες φορές αυτό οφείλεται επειδή υπέβαλαν τον κώδικα εκφώνησης με λίγες μόνο αλλαγές και επομένως πήραν μεγαλύτερο βαθμό από όσο θα έπρεπε.
+
+Οι [[Buff2015](#CITE)] παρουσιάζουν μια αρκετά ολοκληρωμένη σκέψη πάνω στην ιδέα της παροχής αυτοματοποιημένης ανατροφοδότησης. Το σημείο εκκίνησής τους είναι ότι "τα αυτοματοποιημένα συστήματα βαθμολόγησης βοηθούν τους εκπαιδευόμενους να εντοπίζουν τα σφάλματα μέσα τον κώδικα τους, [αλλά] μπορεί ταυτόχρονα να αποθαρρύνουν τους εκπαιδευόμενους από το να σκέφτονται κριτικά και να κάνουν ενδελεχείς ελέγχους, και αντ 'αυτού να ενθαρρύνουν την εξάρτηση από τα τεστ του δασκάλου". Ένα από τα καίρια ζητήματα που εντοπίστηκαν είναι ότι ο εκπαιδευόμενος μπορεί να πράγματι να ελέγξει διεξοδικά τον κώδικα του, αλλά η τελική υλοποίηση μπορεί να μην υπακούει στις προδιαγραφές του εκπαιδευτικού. Σε αυτή την περίπτωση, η "αποτυχία" δεν οφείλεται στην έλλειψη δοκιμών, αλλά στην παρανόηση των απαιτήσεων, και είναι αρκετά απίθανο η εφαρμογή επιπλεόν δοκιμών να εκθέσουν το πρόβλημα. Εάν το σύστημα αυτόματης αξιολόγησης δεν παρέχει διορατική, χρήσιμη και πρακτική ανατροφοδότηση, είναι πολύ πιθανό η συνολική εμπειρία να αποθαρρύνει τον εκπαιδευόμενο.
```

Comment:
να πράγματι να--> πράγματι να
να εκθέσουν--> να εκθέσει

Reply:
Τhanks @pdelias ! Fixed in a new commit.